### ISSUE TITLE
Prevent users from creating categories named "Other"

### ISSUE THREAD

**Issue by nicpottier** [nyaruka/floweditor#845]
Validation fails in this case but we should be friendlier about it in the editor.

**Comment by ericnewcomer** [comment]
It doesn't look like validation actually fails in this case. @rowanseymour should it be?

**Comment by rowanseymour** [comment]
As far as the engine is concerned Other is just another category and there's nothing in the engine that says two categories can't have the same name... or two translated category names can't be the same.. does seem desirable to prevent users from creating their own Others..

How about we replace Other with the 🤷 emoji!

**Comment by ericnewcomer** [comment]
Happy to disallow it in the editor, just wanted to confirm what we wanted our validation behavior to be given the original issue.

**Comment by rowanseymour** [comment]
I don't think this is a validation error in the engine and there are probably flows out there which do it... but it's undesirable and the editor should prevent it. Not sure how/what failed to prompt this ticket.

**Comment by ericnewcomer** [comment]
So was thinking on this, how do we want to handle cases where it already is named other? Do we want to special case allowing it for them and only enforce validation for new categories (which adds complexity). Specifically, if they have an extra Other, open the split, then click ok, what should happen?

**Comment by ericnewcomer** [comment]
I'm guessing we need to allow those using it today to continue to use it due to U-Report or any other external dependencies.
